tests/test_app_initializers.py

```diff
@@ -0,0 +1,33 @@
+import falcon
+import falcon.testing as testing
+
+
+class MediaTypeResource:
+    def on_get(self, req, resp):
+        resp.body = 'foo bar'
+
+
+def test_api_media_type_overriding():
```

Comment:
Could we DRY these two test cases a bit? (Potentially with a little help from `pytest.mark.parametrize(...)`)

Reply:
@vytas7 Fixed 👍